.github/actions/after-e2e/action.yml

```diff
@@ -7,3 +7,9 @@ runs:
     - name: Remove storage file
       shell: bash
       run: rm -f ./storage-state/AdminUserState.json
+
+    - name: Debug Ingress Controller Logs
+      shell: bash
+      if: always()
+      run: |
+        kubectl logs -n ingress-nginx -l app.kubernetes.io/name=ingress-nginx --tail=-1
```

Comment:
can this action be removed and move this debug in the main workflow as a single step?
I guess that `remove storage file` is not really doing anything useful (the workspace is discarded anyway at the end of the job)

Reply:
Done